app/Http/Controllers/CategoriaController.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Categoria;
+
+class CategoriaController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $categorias =Categoria::all();
```

Comment:
Change this to one line
```suggestion
        return Categoria::all();
```